pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -182,35 +268,70 @@ var _ backupManifestFileIterator = &sstFileIterator{}
 // the first level are instead used to extend the current rightmost span in
 // if its current data size plus that of the new span is less than the target
 // size.
-func makeSimpleImportSpans(
```

Comment:
I wonder if we should add the new one along-side, instead of modifying makeSimple in-place? 

I'm just thinking what we might find useful if there are any hard to explain perf changes, in particular the ability to flip back and forth between the covering generators? we could have a setting/env var that just calls makeSimple to materialize the whole list, then just feeds them into the channel so the API matches?

Reply:
Done, keeping makeSimpleImportSpans added a cluster setting to toggle to it.